.idea/codeStyles/Project.xml

```diff
@@ -1,31 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <AndroidXmlCodeStyleSettings>
```

Comment:
Thanks! Can you fix the code style? 👀 
git checkout master .idea/codeStyles/Project.xml